apptools/lru_cache/lru_cache.py

```diff
@@ -98,15 +98,15 @@ def get(self, key, default=None):
 
     def items(self):
         with self._lock:
-            return self._cache.items()
+            return list(self._cache.items())
```

Comment:
Similar to an earlier question regd H5DictNode, should the items/keys/values methods return iterables on Python 3 or should they return lists?

Reply:
I want to say they should return iterables, but the locking presents a thorny problem. Perhaps the safest solution would be to implement this as an iterator (explicitly `yield`-ing items) on Python 3, and using the old/existing implementation on Python 2.

Reply:
I'd prefer leaving this change as-is, mainly because we're interested in removing `LRUCache`. Is that okay?